README.md

```diff
@@ -183,6 +183,11 @@ If you find a security issue with our libraries or services please report it to
     </array>
 ```
 
+6. Add a new keychain group to project Capabilities `com.microsoft.adalcache`
```

Comment:
Thanks for adding this. I agree that it makes a lot of sense. I'd also prefer to link the detailed information here that explains why it is necessary. We have such information in our Wiki: [https://github.com/AzureAD/microsoft-authentication-library-for-objc/wiki/Keychain-on-iOS](https://github.com/AzureAD/microsoft-authentication-library-for-objc/wiki/Keychain-on-iOS).

Reply:
Updated =)